src/app/work-item/work-item.component.ts

```diff
@@ -60,7 +60,7 @@ export class WorkItemComponent implements OnInit, AfterViewInit {
     if (this.loggedIn) {
       this.filters.push({
         id:  1,
-        name: 'Assign to Me',
+        name: 'Assign to me',
```

Comment:
@SMahil no, you have changed it to `Assigned to Me` and not `Assigned to me`. See [here](https://github.com/fabric8io/fabric8-planner/pull/891/files#diff-23889667b50a369926f3bc2f63e68311R63)